src/commands/functions/create.js

```diff
@@ -154,9 +154,9 @@ async function pickTemplate() {
   function filterRegistry(registry, input) {
     const temp = registry.map(x => x.name + x.description)
     const filteredTemplates = fuzzy.filter(input, temp)
-    const filteredTemplateNames = filteredTemplates.map(x => (input ? x.string : x))
+    const filteredTemplateNames = new Set(filteredTemplates.map(x => (input ? x.string : x)))
```

Comment:
Performance optimization.
When the set is being used only once after creation, I am not sure this is actually faster. But this is faster when the set is in the top-level scope and re-used.
[See rule documentation](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/prefer-set-has.md).